pp/integrations/plugin-packs/procedures/operatingsystems-freebsd-snmp.md

```diff
@@ -2,48 +2,854 @@
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **FreeBSD SNMP** brings a host template:
+
+* **OS-FreeBSD-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="OS-FreeBSD-SNMP-custom" label="OS-FreeBSD-SNMP-custom">
+
+| Service Alias | Service Template              | Service Description                                                                                                                                |
+|:--------------|:------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cpu           | OS-FreeBSD-Cpu-SNMP-custom    | Check the rate of utilization of CPU for the machine. This check can give the average CPU utilization rate and the rate per CPU for multi-core CPU |
+| Load          | OS-FreeBSD-Load-SNMP-custom   | Check the server load average                                                                                                                      |
+| Memory        | OS-FreeBSD-Memory-SNMP-custom | Check the rate of the utilization of memory                                                                                                        |
+| Swap          | OS-FreeBSD-Swap-SNMP-custom   | Check virtual memory usage                                                                                                                         |
+
+> The services listed above are created automatically when the **OS-FreeBSD-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                            | Service Description                                                                                                    | Discovery |
+|:---------------------|:--------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|:---------:|
+| Disk-Generic-Id      | OS-FreeBSD-Disk-Generic-Id-SNMP-custom      | Check the rate of free space on the disk. For each checks the name of the disk will appear                             |           |
+| Disk-Generic-Name    | OS-FreeBSD-Disk-Generic-Name-SNMP-custom    | Check the rate of free space on the disk. For each checks the mount pont of the disk will appear                       |           |
+| Disk-Global          | OS-FreeBSD-Disk-Global-SNMP-custom          | Check the rate of free space on disks. For each checks the mount point of disks will appear                            |     X     |
+| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Check access disk of the disk. For each check the name of the disk will appear "label" rather than the letter assigned |           |
```

Comment:
```suggestion
| Disk-IO              | OS-FreeBSD-Disk-IO-SNMP-custom              | Check disk access. For each check the name of the disk will appear as a "label" rather than the letter assigned |           |
```

Reply:
as a label?